AI/pyneogame/Agent/DeepQAgent.py

```diff
@@ -166,3 +170,16 @@ def replay_experience(self, batch_size=64, epochs=30):
                                      validation_split=0.10
                                      )
         return history
+    
+    def get_entry(self):
+        return self.state_size
+    
+    def get_action_size(self):
+        return len(self.actions)
+    
+    def input_model(self, model):
```

Comment:
Take the model as input to the constructor instead and use the same model creation pattern as done in dqn_test.py